sdk/containerregistry/container-registry/samples/v1/javascript/package.json

```diff
@@ -1,31 +1,30 @@
 {
   "name": "azure-container-registry-samples-js",
   "private": true,
-  "version": "0.1.0",
+  "version": "1.0.0",
   "description": "Azure Container Registry client library samples for JavaScript",
   "engine": {
-    "node": ">=8.0.0"
+    "node": ">=12.0.0"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Azure/azure-sdk-for-js.git",
     "directory": "sdk/containerregistry/container-registry"
   },
   "keywords": [
-    "Azure",
-    "Node.js",
-    "JavaScript"
+    "azure",
+    "cloud",
+    "typescript"
```

Comment:
I'm not sure what the precedent for JS/TS libraries is here, but would it make sense to keep JavaScript in the list?

Reply:
I'd prefer JavaScript over typescript because our customers are JS developers and TS is one of the tool in JS ecosystem.  
Regarding the casing, this file is generated.  @willmtemple we should use lower case keywords to be consistent with out SDK libraries?

Reply:
ah I miss read the comments.  It's reasonable to have `JavaScript` in the JS samples' package.json.  @willmtemple maybe you can replace `"typescript"` with `"javascript"` while generate the package.json?

Reply:
My thought is that putting "javascript" as a keyword in an NPM package is a bit like putting ".NET" as a tag on a NuGet package. It's implicitly assumed that an NPM package is a JavaScript package. In any case, these aren't indexed at all, so if we want to replace them I'm happy to add an item to implement that feature, but it seems low priority to me.